scripts/zones/Garlaige_Citadel/npcs/no_name_0.lua

```diff
@@ -0,0 +1,35 @@
+-----------------------------------
+-- Area: Garlaige Citadel
+--  NPC:
+-- !pos -176.759 -1.249 71.511 200
+-----------------------------------
+local ID = require("scripts/zones/Garlaige_Citadel/IDs");
```

Comment:
Can kill the semi-colon!

Reply:
Sorry >.< I copied and paste from another script, I killed it good lol all good now